CHANGELOG.md

```diff
@@ -48,6 +48,12 @@ _None._
 
 _None._
 
+## 5.1.0
+
+### New Features
+
+- `AndroidManifest`: Merged Manifest Diff [#164]
```

Comment:
Can we maybe provide a more explicit entry for this CHANGELOG after all? Doesn't feel super descriptive and kinda abstract to me

Reply:
👍 and how about this (9646fa989b0e59320f39bdd860ab3e996e0b93bc) @AliSoftware ? 🙏